front/app/components/PostShowComponents/Title.tsx

```diff
@@ -38,7 +38,7 @@ const PostTitle = memo<Props>(
     align = 'center',
   }) => {
     return (
-      <Container className={className} align={align}>
+      <Container className={className} align={align} id="e2e-idea-title">
```

Comment:
We still have a duplicate id now when the component from the module renders. The id should be removed from there as well (in the `PostShowTranslatedTitle` component).